index.html

```diff
@@ -2,9 +2,16 @@
 <html>
 <head>
   <title></title>
+  <script src="main.js"></script>
 </head>
 <body>
-
-  <script src="main.js"></script>
+    <form>
+        <fieldset>
+        <legend> Enter your reservation name:</legend>
+        <input id="input" name="userInput" type="text" />
+        <button type="button"  onclick="claimReservation(userInput)">submit</button>
+        <h1 id="live">here</h1>
```

Comment:
what does "live" mean? use indicative names for your IDs, classes, and variables